span-normalizer/span-normalizer/src/main/java/org/hypertrace/core/spannormalizer/jaeger/JaegerSpanToLogRecordsTransformer.java

```diff
@@ -0,0 +1,82 @@
+package org.hypertrace.core.spannormalizer.jaeger;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.google.protobuf.util.Timestamps;
+import io.jaegertracing.api_v2.JaegerSpanInternalModel;
+import io.jaegertracing.api_v2.JaegerSpanInternalModel.Span;
+import java.nio.ByteBuffer;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.apache.kafka.streams.KeyValue;
+import org.apache.kafka.streams.kstream.Transformer;
+import org.apache.kafka.streams.processor.ProcessorContext;
+import org.hypertrace.core.datamodel.Attributes;
+import org.hypertrace.core.datamodel.LogEvent;
+import org.hypertrace.core.datamodel.LogEvents;
+import org.hypertrace.core.spannormalizer.util.AttributeValueCreator;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class JaegerSpanToLogRecordsTransformer
+    implements Transformer<byte[], PreProcessedSpan, KeyValue<String, LogEvents>> {
+
+  private static final Logger LOG =
+      LoggerFactory.getLogger(JaegerSpanToLogRecordsTransformer.class);
+
+  @Override
+  public void init(ProcessorContext context) {
+    // no-op
+  }
+
+  @Override
+  public KeyValue<String, LogEvents> transform(byte[] key, PreProcessedSpan preProcessedSpan) {
+    Span value = preProcessedSpan.getSpan();
```

Comment:
Instead of `buildLogEventRecords`, wrap this method in try/catch and return `null`. As in the current case, we are returning a new value `KeyValue(null, null)`,  and this will be pass to follow down the processor.